default.nix

```diff
@@ -114,12 +114,4 @@ let
     else throw "lock file '${lockFilePath}' has unsupported version ${toString lockFile.version}";
 
 in
-  rec {
-    defaultNix =
-      result
-      // (if result ? defaultPackage.${system} then { default = result.defaultPackage.${system}; } else {});
-
-    shellNix =
-      defaultNix
-      // (if result ? devShell.${system} then { default = result.devShell.${system}; } else {});
-  }
+result
```

Comment:
Doesn't this break all the `shell.nix` and `default.nix` files that depend on these?

Reply:
Also, I'd like to keep any logic (like `defaultPackage.${system}`) in `flake-compat`, rather than in `default.nix` and `shell.nix`. That way we don't have to change the latter if the flake format changes.